app/src/main/java/org/thoughtcrime/securesms/ui/color/Colors.kt

```diff
@@ -0,0 +1,225 @@
+    override val isLight = false
+    override val danger = dangerDark
+    override val disabled = disabledDark
+    override val background = Color.Black
+    override val backgroundSecondary = classicDark1
+    override val text = Color.White
+    override val textSecondary = classicDark5
+    override val borders = classicDark3
+    override val textBubbleSent = Color.Black
+    override val backgroundBubbleReceived = classicDark2
+    override val textBubbleReceived = Color.White
+    override val buttonFilled = primary
+    override val buttonOutline = primary
+    override val qrCodeContent = background
+    override val qrCodeBackground = text
+}
+
+data class ClassicLight(override val primary: Color = primaryGreen): Colors {
+    override val isLight = true
+    override val danger = dangerLight
+    override val disabled = disabledLight
+    override val background = Color.White
+    override val backgroundSecondary = classicLight5
+    override val text = Color.Black
+    override val textSecondary = classicLight1
+    override val borders = classicLight3
+    override val textBubbleSent = Color.Black
+    override val backgroundBubbleReceived = classicLight4
+    override val textBubbleReceived = classicLight4
+    override val buttonFilled = primary
+    override val buttonOutline = Color.Black
+    override val qrCodeContent = text
+    override val qrCodeBackground = backgroundSecondary
+}
+
+data class OceanDark(override val primary: Color = primaryBlue): Colors {
+    override val isLight = false
+    override val danger = dangerDark
+    override val disabled = disabledDark
+    override val background = oceanDark2
+    override val backgroundSecondary = oceanDark1
+    override val text = Color.White
+    override val textSecondary = oceanDark5
+    override val borders = oceanDark4
+    override val textBubbleSent = Color.Black
+    override val backgroundBubbleReceived = oceanDark4
+    override val textBubbleReceived = oceanDark4
+    override val buttonFilled = primary
+    override val buttonOutline = text
+    override val qrCodeContent = background
+    override val qrCodeBackground = text
+}
+
+data class OceanLight(override val primary: Color = primaryBlue): Colors {
+    override val isLight = true
+    override val danger = dangerLight
+    override val disabled = disabledLight
+    override val background = oceanLight7
+    override val backgroundSecondary = oceanLight6
+    override val text = oceanLight1
+    override val textSecondary = oceanLight2
+    override val borders = oceanLight3
+    override val textBubbleSent = oceanLight1
+    override val backgroundBubbleReceived = oceanLight4
+    override val textBubbleReceived = oceanLight1
+    override val buttonFilled = text
+    override val buttonOutline = oceanLight1
+    override val qrCodeContent = text
+    override val qrCodeBackground = backgroundSecondary
+}
+
+val Colors.slimOutlineButton: Color get() = text
```

Comment:
This doesn't seem right. The contentDescription should apply to the button, not the image above. In fact the image should have no content description so that the screen reader reads the button overall.

Reply:
it adds more semantic meaning to usages.

1. it adds easier changes in the future if default color for slim outline changes.
2. Devs can more simply know the color of a slim outline button as its well named and easy to find, even if it is an alias.

Reply:
I'm not sure this makes it easier. If we know from the designs that buttons colors is set to text, then that is what you'd want to see in the code as well.
Especially as we are trying to clean up the themes to match the figma terms.

Reply:
gone